### DISCUSSION TITLE
Feature discussion: add functions similar to `polynomial.m` that generates polynomials with alternative monomial basis

### DISCUSSION THREAD

**Discussion by htadashi** [yalmip/YALMIP#1305] [General discussion]
Would it be beneficial to add functions in YALMIP that generates other monomial basis functions (such as the [scaled monomial basis](https://juliaalgebra.github.io/MultivariateBases.jl/stable/#MultivariateBases.ScaledMonomialBasis))?

**Comment by johanlofberg** [comment]
Not anything I have personal use of, but feel free to contribute. Should be easily done with a 4th argument and then multiplying c in the end with the scaling. The exponents on the monomials in v are available with ` exponents(v,x)`
